stackslib/src/net/server.rs

```diff
@@ -474,7 +485,7 @@ impl HttpPeer {
         let msgs = match convo.chat(node_state) {
             Ok(msgs) => msgs,
             Err(e) => {
-                debug!(
+                info!(
```

Comment:
How often will this log line show up in a node?

Reply:
It hasn't shown up at all on my production node (over the last 18.6 hours).